tests/conftest.py

```diff
@@ -86,7 +86,7 @@ def pytest_addoption(parser):
 
     parser.addoption(
         "--size",
-        choices=["tiny", "small", "large"],
+        choices=["tiny", "small", "large", "mnist"],
```

Comment:
Might want to rename it to `--dataset` or `--dataset-type` (or `--datadir-type` to be future proof?)